src/components/CarbonOffsetProjectDetail.jsx

```diff
@@ -0,0 +1,140 @@
+import React, {useState} from 'react';
+import {useParams, useHistory} from 'react-router-dom';
+import {Row, Col, Card, Button} from 'react-bootstrap';
+import { useInterval } from '../utils/useInterval';
+import './CarbonOffsetProjectDetail.scss'
+
+const CarbonOffsetProjectDetail = () => {
+    const [zipCode, setZipCode] = useState('94117');
```

Comment:
Ah I forgot that we need to keep track of the zip code. We can grab that from the details about the Utility Account that comes back. I can add a server endpoint that the client can poll for this data. It might take 20-30 seconds to be populated though. The zip is entered in Utility Connect but UC doesn't offer a callback with that data...